x/gov/client/cli/tx.go

```diff
@@ -61,35 +59,92 @@ func NewTxCmd(propCmds []*cobra.Command) *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	// TODO Add CLI for new submit proposal
-	// https://github.com/cosmos/cosmos-sdk/issues/10952
-	cmdSubmitProp := NewCmdSubmitProposal()
+	cmdSubmitLegacyProp := NewCmdSubmitLegacyProposal()
 	for _, propCmd := range propCmds {
 		flags.AddTxFlagsToCmd(propCmd)
-		cmdSubmitProp.AddCommand(propCmd)
+		cmdSubmitLegacyProp.AddCommand(propCmd)
 	}
 
 	govTxCmd.AddCommand(
 		NewCmdDeposit(),
 		NewCmdVote(),
 		NewCmdWeightedVote(),
-		cmdSubmitProp,
+		NewCmdSubmitProposal(),
+		cmdSubmitLegacyProp,
 	)
 
 	return govTxCmd
 }
 
-// NewCmdSubmitProposal implements submitting a proposal transaction command.
+// NewCmdSubmitLegacyProposal implements submitting a proposal transaction command.
```

Comment:
Why is this called `NewCmdSubmitLegacyProposal?

Reply:
So before:
- `submit-proposal`: submits a content-based proposal

Now:
- `submit-proposal`: submits a Msg-based proposal
- `submit-legacy-proposal`: submits a content-based proposal

This `NewCmdSubmitLegacyProposal` is for the second one

Reply:
> Is it necessary to support backwards compatibility via the CLI?

I have a slight preference for keeping `submit-legacy-proposal`. The current PR breaks user scripts as you say, but a `s/submit-proposal/submit-legacy-proposal` will fix those scripts.

If we only allowed the new Msg-based proposals, there will be some non-trivial changes to make for CLI users (aka converting their JSONs)

Reply:
Oh wait, I just noticed this comment is wrong, and that was what you were referring to... 🤦 
```suggestion
// NewCmdSubmitProposal implements submitting a proposal transaction command.
```